src/icons/SwapArrows.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 }
 
 const SwapArrows = ({ size = 24, color = Colors.black }: Props) => (
-  <Svg width={size} height={size} viewBox="0 0 24 24" fill="none">
```

Comment:
what's the reason for this not being a square?

I think that having all the icons squared doesn't distort their initial proportions and enables them to be easily reusable in other parts of the app.

note how all the icons in the [figma components library](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=317-135&mode=design&t=2wyGE58IAneSuv2f-0) are squared:
![image](https://github.com/valora-inc/wallet/assets/2737872/2cb9f0e4-2d01-49ab-abc2-d4ab53e6cced)

Reply:
Ahhh good catch! If you double click on the icons there, it creates a smaller bounding box that directly touches the SVG border, which is what I think I used when exporting the icons.